src/extension/_locales/de/messages.json

```diff
@@ -419,12 +419,12 @@
   "error_logout": {
     "message": "Die Abmeldung ist fehlgeschlagen. Bitte versuchen Sie es später erneut."
   },
-  "error_preview_4XX": {
-    "message": "Das Generieren der Vorschau ist fehlgeschlagen. Überprüfen Sie die Details für weitere Informationen."
-  },
   "error_preview": {
     "message": "Das Generieren der Vorschau ist fehlgeschlagen. Bitte versuchen Sie es später erneut."
   },
+  "error_preview_4XX": {
+    "message": "Erstellung der Vorschau fehlgeschlagen. Weitere Informationen finden Sie in den Details."
```

Comment:
This new translation of "Preview generation failed" is different compared to existing other strings that start with the same sentence (e.g. line 423). We should keep all occurrences of this sentence the same across a language file.